src/templates/ProductPage.js

```diff
@@ -1,16 +1,41 @@
-import React from 'react'
+import React, { useContext, useEffect, useState } from 'react'
 import { graphql, withPrefix } from 'gatsby'
 
 import SEO from '../components/SEO'
 import Photo from '../components/Photo'
 import Badge from '../components/Badge'
 import AddToCart from '../components/AddToCart'
+import { Shopkit } from '../shopkit'
 
 function ProductPage({ data: { product } }) {
+  const { moltin } = useContext(Shopkit)
+  const [inventoryAvailability, setInventoryAvailability] = useState(0)
+  const [inventoryLoading, setInventoryLoading] = useState(true)
+  const [inventoryError, setInventoryError] = useState(false)
+
+  useEffect(() => {
+    async function getProductInventory() {
+      try {
+        const {
+          data: { available }
+        } = await moltin.get(`inventories/${product.id}`)
+
+        setInventoryLoading(false)
+        setInventoryAvailability(available)
```

Comment:
@ynnoj what if `manage_stock` is set to false?

Reply:
Does that not create an associated inventory record?

Reply:
https://deploy-preview-18--gatsby-demo-store.netlify.com/products/crown has `manage_stock` set to `false`, yet it still shows as out of stock and prevents me from adding to cart.

Reply:
I thought if `manage_stock` was set to false on creation then no inventory record was created. In this instance, there is an inventory record.

I guess we'll have to check `manage_stock` on the product.

Reply:
What if I set it to false because I no longer wish to manage stock?